src/Components/WebAssembly/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -129,6 +129,7 @@
       <_BlazorOutputWithTargetPath Include="@(_BlazorResolvedAssembly)">
         <BootManifestResourceType Condition="'%(Extension)' == '.dll'">assembly</BootManifestResourceType>
         <BootManifestResourceType Condition="'%(Extension)' == '.pdb'">pdb</BootManifestResourceType>
+        <BootManifestResourceType Condition="@(BlazorLazyLoad->AnyHaveMetadataValue('Identity', '%(Filename)')) != ''">dynamicAssembly</BootManifestResourceType>
```

Comment:
Should this be `BlazorWebAssemblyLazyLoad`?

Reply:
Yes. We should make sure that the syntax indicates that this will only apply to Blazor WASM.

Do you know if there is a way to configure autocomplete for special properties like this? I opted for `BlazorLazyLoad` in the interest of brevity.

Reply:
> Do you know if there is a way to configure autocomplete for special properties like this? I opted for BlazorLazyLoad in the interest of brevity.

I recall @DamianEdwards pointed us to the schema for this once, but I can't seem to find it in my emails.